src/Utility/ResponseToArrayHelper.php

```diff
@@ -87,6 +87,11 @@ protected function responseToArray(ResponseInterface $response, bool $expandComp
      */
     protected function normalizeExpandFalseForHybrid(array $responseArray): array
     {
+        // If empty, no further processing is needed.
+        if (empty($responseArray)) {
+            return  $responseArray;
```

Comment:
an extra space between return and the variable.

Reply:
Fixed!